test/sequential/rewardsv2.poolLiquidityRewards.test.ts

```diff
@@ -2,7 +2,6 @@
  *
  * @group rewardsV2Sequential
  */
-
 import { getApi, getMangataInstance, initApi } from "../../utils/api";
```

Comment:
Should we add { jest } and `jest` function here?

Reply:
looks like is not necessary, there is no usage for such. will keep an eye  O_o